woff1/tools/validator/validator.py

```diff
@@ -9,6 +9,7 @@
 
 # import
 
+from __future__ import division, print_function
```

Comment:
Ensure that division using the `/` operator and `print` work consistently across Python versions 2 and 3.